### ISSUE TITLE
Not correctly formatting error when using it with nightwatch-html-reporter

### ISSUE THREAD

**Issue by felipeplets** [reallymello/nightwatch-axe-verbose#3]
When using it with nightwatch-html-reporter the report is showing failures as in the following screenshot: 

![Screen Shot 2020-12-29 at 8 10 12 PM](https://user-images.githubusercontent.com/119980/103323649-dd24ad80-4a11-11eb-99b4-bc6f10ebda26.png)

When I would expect the report to show the 2 failures as red X with the message. 

This seems rather to be an error in the html reporter but when using with the standard NightwatchJS API I was getting the correct format, so it seems to be a possible adjustment necessary from either side.

**Comment by reallymello** [comment]
I used to see that issue where failures had a green check as well, but not currently under nightwatch v1.5.1. Can you try on your side and see if it is still happening under the latest version of nightwatch?

![image](https://user-images.githubusercontent.com/18431256/103356016-c87dff00-4a7d-11eb-825c-f3296cf711ed.png)

**Comment by felipeplets** [comment]
Yes, and the terminal actually shows just as yours but the `nightwatch-html-reporter` shows as in mine. It seems like an issue in `nightwatch-html-reporter` when using `verify` instead of `assert`. I will download the reporter source code later today and see if I can fix over there.

**Comment by reallymello** [comment]
I created an assertion called `isAccessible(...)` to use in place of `axeRun(...)` and since it is an assertion instead of a command, like axeRun, it should show the results better in the HTML report.

So instead of something like `browser.axeInject().axeRun()` you'd call `browser.axeInject().assert.isAccessible()` to use it
